pkg/operator/encryption/deployer/revisionedpod.go

```diff
@@ -213,21 +213,26 @@ func nodeNames(pods []*corev1.Pod) sets.String {
 }
 
 func categorizePods(pods []corev1.Pod) (good []*corev1.Pod, bad []*corev1.Pod, progressing bool, err error) {
+	if len(pods) == 0 {
+		return nil, nil, true, err
+	}
```

Comment:
```
return good, bad, len(pods) > 0, nil
```

Reply:
that‘s equivalent. But return early is more Go like, isn‘t it?